recipe/meta.yaml

```diff
@@ -18,6 +19,7 @@ requirements:
   host:
   run:
   run_constrained:
+    - vc14_runtime >=14.29.30037
     - vs2015_runtime >=14.29.30037
```

Comment:
Not sure if this is really necessary anymore. We don't have compilers that old anymore on the vc feedstock; but if we do keep it I think we should ~change to~ add `vc14_runtime`.